cactusbot/handler.py

```diff
@@ -6,15 +6,52 @@
 
 
 class Handlers(object):
-    """Handlers."""
+    """Handles all the events
+
+    For a function to have the ability to be used as an event handler, it must
+    be prefixed with `on_`, and then followed by the event name.
```

Comment:
`s/function/method`.

Also, make a note that it receives `packet` as the single argument.

Because we have a standard for what *kind* of packet is passed, depending on the event, we should mention that, too, in a [table](https://github.com/CactusDev/CactusBot/blob/develop/cactusbot/commands/command.py#L48)!

- `message`: `MessagePacket`
- `follow`, `sub`, `host`: `EventPacket`
- etc.
- all others receive a base `Packet`